src/python-bindings/collector.cpp

```diff
@@ -14,6 +14,12 @@
 
 #include "module_lock.h"
 
+// http://docs.python.org/3/c-api/apiabiversion.html#apiabiversion
+#if PY_MAJOR_VERSION >= 3
+   #define PyInt_Check(op)  PyNumber_Check(op)
+   #define PyString_Check(op)  PyBytes_Check(op)
```

Comment:
Or better, it looks like what the above comment really means is we can use PyBytes_* function names in python2, which are aliases to the PyString_* equivalents.